simplq/src/components/pages/Home/CreateJoinForm.jsx

```diff
@@ -11,7 +11,9 @@ const CreateJoinForm = ({ history }) => {
   const [textFieldValue, setTextFieldValue] = useState('');
   const [invalidMsg, setInvalidMsg] = useState('');
   const [createInProgress, setCreateInProgress] = useState(false);
-  const handleClick = () => {
+  const [joinInProgress, setJoinInProgress] = useState(false);
+
+  const handleCreateClick = () => {
     if (textFieldValue === '') setInvalidMsg('Queue name is required');
     else {
```

Comment:
We should do the isQueueNameValid check for join and create, right?

Reply:
Just checked, the textbox doesn't allow invalid characters. So, for example, if you type "!", it won't go in. So we are good.

Reply:
Oh, got it. The check is there in the handleTextChange function.